### DISCUSSION TITLE
Specific woocommerce category template

### DISCUSSION THREAD

**Discussion by Daxpnx** [bootscore/bootscore#463] [Q&A]
Hi,
Can you tell me if it's possible to create a custom template for only one woocommerce products category ?
Thank you,
Damien

**Comment by crftwrk** [comment]
Hi Damien,

unfortunately this is not possible because theme uses `woocommerce.php`. Using this file does not allow to edit `archive-product.php`.  But there are several workarounds, check here https://github.com/orgs/bootscore/discussions/457.

**Comment by Daxpnx** [comment]
Thank you for the response.
My problem is that I want to change the content of the loop (I think it's woocommerce_content() ? )
Isn't it possible ?

**Reply by crftwrk** [reply]
Loop is the `content-product.php` https://github.com/bootscore/bootscore/blob/main/woocommerce/content-product.php, you can simply override file im your child.

**Comment by Daxpnx** [comment]
Ok, thank you.
For my specific category I do this in woocommerce.php
Not very clean .. :/

```
$term_id = get_queried_object()->term_id;
if ($term_id == 22) {
    include('panier-local.php');
}
else 
    woocommerce_content(); 
```
